src/js/components/SearchPage.jsx

```diff
@@ -433,31 +433,23 @@ const SearchResult = ({ item, CTA, onPick }) => {
             }
             data-id={cid}
         >
-            <a
-                href={charityUrl}
-                onClick={onClick}
-                className="logo col-md-2 col-xs-4"
-            >
-                {item.logo ? (
+            {item.logo && (
```

Comment:
why is this '&&' not '?' ?

Reply:
Its a little more concise and removes the need to specify null.. The other option would be  `sth ? Component : null`